kalite/topic_tools/__init__.py

```diff
@@ -156,14 +175,12 @@ def get_exercise_cache(force=False, language=settings.LANGUAGE_CODE):
                         available = True
                 exercise["all_assessment_items"] = items
             else:
-                available = os.path.isfile(TEMPLATE_FILE_PATH % exercise_template)
+                available = exercise_template in exercise_templates
 
                 # Get the language codes for exercise templates that exist
                 # Try to minimize the number of os.path.exists calls (since they're a bottleneck) by using the same
                 # precedence rules in i18n.select_best_available_languages
```

Comment:
Yeah, you're right, I was thinking of a different PR and thoughtlessly linked to another one.